src/lib.rs

```diff
@@ -63,17 +64,21 @@
 //! TODO: write a small example that uses clap derive
 //!
 
-use std::io::IsTerminal;
+use std::{
+    io::{self, IsTerminal, Write},
+    sync::Arc,
+};
 
-use colored::Colorize;
 use log::SetLoggerError;
+use termcolor::{BufferWriter, Color, ColorChoice, ColorSpec, WriteColor};
 
 pub const MODULE_PATH_UNKNOWN: &str = "?";
 pub const MODULE_LINE_UNKNOWN: &str = "?";
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone)]
 pub struct Logger {
     level: log::Level,
+    writer: Arc<BufferWriter>,
```

Comment:
I'm not sure if it's better to wrap the `BufferWriter` in an `Arc` or to remove the `Clone` derive on the logger struct and let the end user use an `Arc` if they want to share the logger between threads.

Reply:
the `Arc` on the `BufferWriter` seems good to me 👍